java/arcs/core/storage/ReferenceModeStore.kt

```diff
@@ -155,25 +155,31 @@ class ReferenceModeStore private constructor(
     @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
     val backingStore = DirectStoreMuxer<CrdtData, CrdtOperation, Any?>(
         storageKey = backingKey,
-        backingType = backingType,
-        callbackFactory = { muxId ->
-            ProxyCallback { message ->
-                CoroutineScope(coroutineContext).launch {
-                    receiveQueue.enqueue(
-                        PreEnqueuedFromBackingStore(message.toReferenceModeMessage(), muxId)
-                    )
-                }
-            }
-        }
+        backingType = backingType
     )
 
+    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+    val backingStoreId: Int
+    val containerStoreId: Int
```

Comment:
Is this also intended to be VisibleForTesting-only? If so.. it should be commented/annotated as well. Otherwise, it should probably be private.

Reply:
Good catch, I've made it private